package/MDAnalysis/core/universe.py

```diff
@@ -394,17 +389,32 @@ def trajectory(self, value):
 
     def add_TopologyAttr(self, topologyattr):
         self._topology.add_TopologyAttr(topologyattr)
-        # Add the Attr to the Group (AtomGroup, RG, SG)
-        self._Group._add_prop(topologyattr)
-        # Add the Attr to the component (Atom, Residue, Segment)
+        self._process_attr(topologyattr)
+
+    def _process_attr(self, attr):
+        """Squeeze a topologyattr for its information
+
+        Grabs:
+         - Group properties (attribute access)
+         - Component properties
+         - Transplant methods
+        """
+        self._Group._add_prop(attr)
+
         try:
```

Comment:
I didn't know if
 - every Attribute **must** have a .level attribute
 - every Attribute's .level would correspond to either Atom/Residue/Segment (if `bond` becomes an available level for instance)

We should make a wiki page on Attribute API